### PR TITLE
Ovhapi lib

### PR DESCRIPTION
**Pull request by bonidier**
Hi @denouche,

I've been inspired by your **ovh-api-bash-client** usage on [ovh-dyndns](https://github.com/denouche/ovh-dyndns)

I've decided to build a set of libs embedded into ovh-api-bash-client repo to avoid duplicating some common functions.
Also, externals scripts don't have to know that JSON.sh is needed.

External scripts should now just include a single file to use wrapper to ovh-api-bash-client and play with JSON result. 

This was a very hard work, with some drafts (see commits), and now it's done for me.

It's fully documented, I let you play with all this things :-)

Regards,
Didier

### CODE REVIEW

**Comment by denouche** [comment]
Hi! 
Thank you for your pull request, coming back from holidays, I will check it soon!

File: contrib/jsonsh-lib.sh [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    local wanted_key=$2
+
+    if [[ "${action}" == "getValue" ]] && [[ -z "${wanted_key}" ]]; then
+      echo "key is required" >&2
+      exit 1
+    fi
+
+    JSONSH_SEPARATOR=${JSONSH_SEPARATOR:-":"}
+
+    awk -F '\t' \
```
Comment (denouche) [inline contrib/jsonsh-lib.sh:105]: I love this one :)

**Comment by bonidier** [comment]
Hi @denouche,
Any news about your check ?

Regards

**Comment by denouche** [comment]
Yes, sorry for the latency.

Really nice contribution with a lot of examples and wrappers, thank you!

**Pull request merged**